assets/js/base/hooks/payment-methods/constants.js

```diff
@@ -0,0 +1,8 @@
+export const STATUS = {
+	PRISTINE: 'pristine',
+	STARTED: 'started',
+	FINISHED: 'finished',
+	ERROR: 'has_error',
+	FAILED: 'failed',
+	SUCCESS: 'success',
```

Comment:
> success: (select.isSuccessful or dispatch.success(). Indicates the payment processing succeeded. I'm assuming the checkout would use this state to trigger redirect to the success url.

I ❤️ that the gateway is in control of when the payment is considered finished. That will simplify some things in Stripe + SCA :)